hvpy/api_groups/movies/queue_movie.py

```diff
@@ -0,0 +1,143 @@
+        Image scale in arcseconds per pixel.
+    format
+        Movie format (mp4, webm, flv).
+        Default value is `mp4`, optional.
+    frameRate
+        Movie frames per second.
+        Default value is `15` frames per second, optional.
+    maxFrames
+        Maximum number of frames in the movie, can be capped by the server.
+        Default value is `None`, optional.
+    scale
+        Overlay an image scale indicator.
+        Default value is `None`, optional.
+    scaleType
+        Set the image scale indicator.
+        Default value is `None`, optional.
+    scaleX
+        Horizontal offset of the image scale indicator in arcseconds with respect to the center of the Sun.
+        Default value is `None`, optional.
+    scaleY
+        Vertical offset of the image scale indicator in arcseconds with respect to the center of the Sun.
+        Default value is `None`, optional.
+    movieLength
+        movie length in seconds.
+        Default value is `None`, optional.
+    watermark
+        Optionally overlay a Helioviewer.org watermark image.
+        Default value is `True`, optional.
+    width
+        Width of the field of view in pixels. (Used in conjunction width `x0`,`y0`, and height).
+        Default value is `None`, optional.
+    height
+        Height of the field of view in pixels. (Used in conjunction width `x0`,`y0`, and width).
+        Default value is `None`, optional.
+    x0
+        The horizontal offset of the center of the field of view from the center of the Sun. Used in conjunction with `y0`, width, and height.
+        Default value is `None`, optional.
+    y0
+        The vertical offset of the center of the field of view from the center of the Sun. Used in conjunction with `x0`, width, and height.
+        Default value is `None`, optional.
+    x1
+        The horizontal offset of the top-left corner of the field of view with respect to the center of the Sun (in arcseconds).
+        Used in conjunction with `y1`, `x2`, and `y2`.
+        Default value is `None`, optional.
+    y1
+        The vertical offset of the top-left corner of the field of view with respect to the center of the Sun (in arcseconds).
+        Used in conjunction with `x1`, `x2`, and `y2`.
+        Default value is `None`, optional.
+    x2
+        The horizontal offset of the bottom-right corner of the field of view with respect to the center of the Sun (in arcseconds).
+        Used in conjunction with `x1`, `y1`, and `y2`.
+        Default value is `None`, optional.
+    y2
+        The vertical offset of the bottom-right corner of the field of view with respect to the center of the Sun (in arcseconds).
+        Used in conjunction with `x1`, `y1`, and `x2`.
+        Default value is `None`, optional.
+    callback
+        Wrap the response object in a function call of your choosing.
+        Default value is `None`, optional.
+    size
+        Scale video to preset size.
+        Default value is `0`, optional.
```

Comment:
```suggestion
        Default value is 0, optional.
```
I don't think a value needs to be quoted like this.